build/teamcity/cockroach/nightlies/pebble_nightly_race_impl.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
Could you change the name of this file to `pebble_nightly_ycsb_race_impl.sh` so it matches the script that calls it?